src/types.ts

```diff
@@ -38,11 +39,17 @@ export interface FiatConnectApiClient {
   login(params?: LoginParams): Promise<Result<'success', ResponseError>>
   isLoggedIn(): boolean
   createQuoteIn(
-    params: QuoteRequestBody,
+    params: CreateQuoteParams,
   ): Promise<Result<QuoteResponse, ResponseError>>
   createQuoteOut(
-    params: QuoteRequestBody,
+    params: CreateQuoteParams,
   ): Promise<Result<QuoteResponse, ResponseError>>
+  createQuoteInPreview(
```

Comment:
nit: should the preview be `get` instead of `create` since a quoteId is not "created"?